cmd/root.go

```diff
@@ -107,4 +108,12 @@ func init() {
 		"o",
 		"Output file format. One of: "+strings.Join(output.AllowedValues(), ", "),
 	)
+
+	rootCmd.Flags().BoolVarP(
+		&layers,
+		"layers",
+		"s",
+		false,
+		"Add layers per stage (default false)",
```

Comment:
```suggestion
		"display all layers (default false)",
```

Reply:
Two quick notes here:
- I will lowercase all the other help texts once this PR is merged
- Please also run `dockerfilegraph --help` and update the README.md with the new output – thx!

Reply:
Hi patrick, could you provide me the dockerfile used to display the example images in the README.md?

Reply:
It's [this one](https://github.com/danielTobon43/dockerfilegraph/blob/feature-layers-per-stage/example/Dockerfile). Also, feel free to change it as necessary to demonstrate the new feature 🙂

Reply:
> I will lowercase all the other help texts once this PR is merged

Done: #90